resharper/resharper-unity/src/Yaml/Psi/DeferredCaches/AssetHierarchy/AssetDocumentHierarchyElementContainer.cs

```diff
@@ -22,11 +22,12 @@ namespace JetBrains.ReSharper.Plugins.Unity.Yaml.Psi.DeferredCaches.AssetHierarc
     [SolutionComponent]
     public class AssetDocumentHierarchyElementContainer : IUnityAssetDataElementContainer
     {
-        private readonly IPersistentIndexManager myManager;
+        public readonly IPersistentIndexManager Manager;
+        public readonly UnityExternalFilesPsiModule PsiModule;
+        public readonly MetaFileGuidCache MetaFileGuidCache;
+        
```

Comment:
They don't look like they're used externally to the class. Make them `private` again.

Reply:
oops, it was temp change. Thanks :)